scribble-lib/scribble/private/manual-bib.rkt

```diff
@@ -65,7 +65,9 @@
        `(" " ,@(decode-content (list location)) ,(if date "," "."))
        null)
      (if date `(" " ,@(decode-content (list date)) ".") null)
-     (if url `(" " ,(link url (tt url))) null)
+     (if url (list " "
```

Comment:
This is bad...

Reply:
@jackfirth I think this occurs because Resyntax invokes range formatting on only the subexpression, so it doesn't format the entire if expression...

(Actually fmt ideally should collapse `(list " " (link url (tt url)))` into one line, circumventing this issue, but without shadowing, `link` is a special form that should be formatted vertically, so that's why fmt introduces a newline)

Reply:
Yeah, that's definitely the cause. I'm not sure how to fix it from the Resyntax side.